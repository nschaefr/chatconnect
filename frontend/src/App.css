html, body, #root, .app {
	height: 100%;
	width: 100%;
	background-color: #31313A;
}