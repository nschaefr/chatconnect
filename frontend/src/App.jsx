import axios from "axios";
import { ContextProvider } from "./components/utils/user-context";
import Routes from "./Routes.jsx";
/**
 * Landing Page
 */
function App() {
  axios.defaults.baseURL = "http://localhost:4040";
  axios.defaults.withCredentials = true;
  return (
    <ContextProvider>
      <Routes />
    </ContextProvider>
  );
}

export default App;
