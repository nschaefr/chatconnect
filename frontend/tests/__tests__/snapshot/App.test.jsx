import React from "react";
import { render } from "@testing-library/react";
import App from "../../../src/App";

describe("app snapshots", () => {
  test("render app component", () => {
    const { container } = render(<App />);

    expect(container).toMatchSnapshot();
  });
});
