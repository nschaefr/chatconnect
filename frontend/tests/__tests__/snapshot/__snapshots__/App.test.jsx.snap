// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app snapshots render app component 1`] = `
<div>
  <div
    class="page"
  >
    <div
      class="form"
    >
      <div
        class="headerDiv"
      >
        <img
          src="[object Object]"
          width="40px"
        />
        <p
          class="header"
        >
          chatconnect
        </p>
      </div>
      <p
        class="slogan"
      >
        Sign In
      </p>
      <form>
        <input
          class="input"
          data-testid="username-input"
          placeholder="Username"
          type="name"
        />
        <br />
        <input
          class="pwd"
          data-testid="password-input"
          placeholder="Password"
          type="password"
        />
      </form>
      <div
        class="decision"
      >
        <div>
          <p
            class="text"
          >
            You haven't signed up yet?
          </p>
          <p
            class="btn"
            data-testid="create-account"
          >
            Create an account
          </p>
        </div>
        <div
          class="btn2"
        >
          <p
            class="text2"
            data-testid="submit-button"
          >
            Log In
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
